src/tab-container/index.tsx

```diff
@@ -177,10 +182,46 @@ export const TabContainer = factory(function TabContainer({
 		);
 	};
 
-	const content = [
-		<div key="buttons" classes={themeCss.tabButtons}>
-			{tabs.map(renderTab)}
-		</div>,
+	/**
+	 * Computes the height of browser-rendered horizontal scrollbars using a self-created test element.
+	 * May return 0 (e.g. on OS X browsers under default configuration).
+	 */
+	const computeHorizontalScrollbarHeight = (): number => {
```

Comment:
@KaneFreeman we have a middleware (https://github.com/dojo/widgets/blob/master/src/middleware/offscreen.ts) that can be used for offscreen  rendering in order to calculate dimensions call `offscreen`, also I think we can leverage the `getOrSet` icache api to make it a bit neater. I think something like this will work:

```tsx
icache.getOrSet('horizontalScrollbarHeight', () => {
    return offscreen(
        () => <div classes={[themeCss.scrollTest]}/>,
        (node) => node.offsetHeight - node.clientHeight
    );
});
```

Also I don't think the class `themeTest` should be a themeable class, probably should be a fixed class, which means we'll need to add a new css file for the tab-container.

Reply:
Caveat I haven't tried it, but I think it should work.